pcdsdevices/epics_motor.py

```diff
@@ -995,6 +998,75 @@ class BeckhoffAxis(EpicsMotorInterface):
     home_forward = None
     home_reverse = None
 
+    def subscribe(
+        self,
+        callback: Callable,
+        event_type: Optional[str] = None,
+        run: bool = True,
+    ) -> int:
+        """
+        Subscribe to events this event_type generates.
+
+        See ophydobj.subscribe for full details.
+
+        This is a thin wrapper over subscribe for custom error handling
+        to intercept the normal end of move handler.
+
+        Unlike during EpicsMotor's move, the move status will
+        be set to the Beckhoff error message if there is one.
+        The move will be marked as successful otherwise.
+
+        Possibly this was unnecessary with an ophyd PR to allow me to
+        customize the error handling, but without that this seems like
+        the easiest way to sidestep the status._finished call and ignore
+        the error logic in _move_changed.
+        """
+        # Mutate subscribe appropriately if this is the exact sub req done
+        # See PositionerBase.move
+        if all((
+            event_type == self._SUB_REQ_DONE,
+            callback.__qualname__ == 'StatusBase._finished',
```

Comment:
Ahh, to future-proof ourselves in case we or someone else gets around to fixing ophyd to not call _finished any more?

Reply:
Yea exactly.  At the very least it might mitigate the initial confusion of thinking _finished and set_finished were different things when we read this block of code

Reply:
There's actually a subtle difference between the two since _finished might call set_exception, so I think we'll need to update this anyway when ophyd updates, but I don't think there's any harm in future-proofing the simplest update case.